code/modules/antagonists/changeling/powers/gaseous_pores.dm

```diff
@@ -0,0 +1,24 @@
+/datum/action/changeling/gaseous_pores
+	name = "Gaseous Pores"
+	desc = "Our skins bursts, releasing somniferous gases to put opponents to sleep and cover our retreat."
+	helptext = "We are still affected by the gases we emit, internals are recommended. Functions while unconscious."
+	button_icon_state = "smoke"
+	chemical_cost = 35
+	dna_cost = 2
+	req_stat = UNCONSCIOUS
+	var/range = 4
```

Comment:
```suggestion
	var/static/range = 4
```
This should probably just be a define tbh

Reply:
actually do you even need a define or var here, it's not like it would ever change or anything.

Reply:
So do I change it or do i not?